rasa/core/nlg/__init__.py

```diff
@@ -1,3 +1,3 @@
 from rasa.core.nlg.generator import NaturalLanguageGenerator
-from rasa.core.nlg.template import TemplatedNaturalLanguageGenerator
+from rasa.core.nlg.response import TemplatedNaturalLanguageGenerator
```

Comment:
Hmm it talks about generating responses right? In this case, would "responses" be more appropriate? I also renamed template.py to response.py in the same directory.

Reply:
You mean something like `ResponseGenerator`? Sounds good 👍🏻

Reply:
I just searched, there are 57 `TemplatedNaturalLanguageGenerator` in the code, I'll suggest leaving it for now, it will further increase the scope of the PR. How about we also stay with `rasa.core.nlg.response`?

Reply:
Both fine for me 👍🏻